### PR TITLE
Add Ukelele

### PR DESCRIPTION
**Pull request by herrbischoff**
Replaces #207.

### CODE REVIEW

**Comment by herrbischoff** [comment]
No interest?

**Comment by iCHAIT** [comment]
I am note sure about this one, can you confirm that it is indeed an awesome application, as far as releases are concerned it is  quite active and I don't see many similar applications that are already on the list, so that is definitely a plus point for it.

I'll merge this but I guess you need to sort it alphabetically?

**Comment by herrbischoff** [comment]
It's the absolute best keyboard layout editor I have found — and I quite possibly tried them all. I'm going to check the sorting and update the pull request.

**Comment by herrbischoff** [comment]
As my Vim is set to automatically remove trailing whitespace (I get very Sith about that), there was also one correction there. Sorry for the rather not so clean PR.

File: README.md [hunk 1]
```diff
@@ -231,26 +231,27 @@
...
 - [Subtitles](http://subtitlesapp.com/) - Automatically downloads subtitles for your movies and TV shows.
 - [TeamViewer](https://www.teamviewer.com/en-us/index.aspx) - Remotely control another computer.
 - [TextBar](http://www.richsomerfield.com/apps/) - TextBar is a tiny but powerful app that lets you add any text to your MenuBar.
 - [Typora](http://www.typora.io/) - Another minimal Markdown editor.
+- [Ukelele](http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=ukelele) - Unicode Keyboard Layout Editor.
 - [Übersicht](http://tracesof.net/uebersicht/) - Run system commands and display their output on your desktop as widgets. [![Open-Source Software][OSS Icon]](https://github.com/felixhageloh/uebersicht)
 - [The Unarchiver](http://unarchiver.c3.cx/unarchiver) - Unarchive many different kinds of archive files. [![Open-Source Software][OSS Icon]](https://bitbucket.org/WAHa_06x36/theunarchiver)
 - [Wineskin](http://wineskin.urgesoftware.com/) - Run Windows applications and games on your Mac. [![Open-Source Software][OSS Icon]](http://sourceforge.net/p/wineskin/code/ci/master/tree/)
```
Comment (iCHAIT) [inline README.md]: @herrbischoff  `The Unarchiver` should be before `Ukelele`?

File: README.md [hunk 2]
```diff
@@ -231,26 +231,27 @@
...
 - [Knock](http://www.knocktounlock.com) - Unlock your Mac quickly and securely. Now with Apple Watch.
 - [Loading](http://bonzaiapps.com) - See when apps are using your network in your Mac menubar. [![Open-Source Software][OSS Icon]](https://github.com/BonzaiThePenguin/Loading/)
 - [MacDown](http://macdown.uranusjr.com/) - Markdown editor. [![Open-Source Software][OSS Icon]](https://github.com/uranusjr/macdown)
-- [Mackup](https://github.com/lra/mackup) - Keep your application settings in sync. [![Open-Source Software][OSS Icon]](https://github.com/lra/mackup)
 - [MacPass](http://mstarke.github.io/MacPass/) - Password Manager. [![Open-Source Software][OSS Icon]](https://github.com/mstarke/MacPass)
+- [Mackup](https://github.com/lra/mackup) - Keep your application settings in sync. [![Open-Source Software][OSS Icon]](https://github.com/lra/mackup)
 - [Media Converter](http://media-converter.sourceforge.net/) - Simple (drag and drop) but advanced media conversion. [![Open-Source Software][OSS Icon]](http://sourceforge.net/p/media-converter/code/HEAD/tree/)
```
Comment (iCHAIT) [inline README.md]: Shouldn't `Mackup` be placed before `MacPass`?

File: README.md (hunk 1 above)
Comment (herrbischoff) [inline README.md]: Nope. The last time I checked, `k` came before `n` in the alphabet. ;P The only debatable positioning would be `Übersicht` vs. `The Unarchiver`.

File: README.md (hunk 2 above)
Comment (herrbischoff) [inline README.md]: There you are right. Ahh... Vim sorts uppercase before lowercase. Should have known better.

File: README.md (hunk 1 above)
Comment (iCHAIT) [inline README.md]: I am sorry for all this confusion but I guess here we are talking about `The Unarchiver` and `Ukelele` which deals with a `t` and an `u`.

**Comment by dkhamsing** [comment]
can you rebase/fix the conflicts?

**Comment by herrbischoff** [comment]
Closing in favor of #337.

**Pull request closed without merging**